app/src/components/Week.js

```diff
@@ -34,14 +64,22 @@ const Week = (data) => {
         </button>
       </h2>
       <CardGroup>
-        {dates.map((date, i) => (
-          <DayOfWeek
-            date={date}
-            key={i}
-            svg={svgs[i]}
-            data={data.data.phase[dates[i].date()].phaseName}
-          />
-        ))}
+        {dates.map((date, i) => {
+          const tasksOfTheDay = dayTasks.filter(
+            (task) =>
+              dayjs(task.date).format("YYYY-MM-DD") ===
+              date.format("YYYY-MM-DD"),
+          );
+          return (
+            <DayOfWeek
+              date={date}
+              tasks={tasksOfTheDay}
+              key={i}
+              svg={data?.[dates[i].date() - 1]?.svg}
+              data={data?.[dates[i].date() - 1]?.AlexPhase}
```

Comment:
This is a neat way to sorta error handle/check object props! 👀 Never knew that this was a thing